crates/lumina_server/src/lobby/in_game.rs

```diff
@@ -52,6 +54,9 @@ fn start_game(
                 &room_manager,
             );
 
+            // Trigger spaceship reset
+            evw_reset_spaceships.send(ResetSpaceships);
```

Comment:
You need to specify the room id for the spaceship resets xd otherwise if there's more than 1 room, the other room's spaceship will also reset health hahaha